TVC_MVP/CountyPresenter.swift

```diff
@@ -92,14 +96,12 @@ class CountyPresenter: CountyPresenterProtocol {
 
     private func generateAlertMessage(county: County) -> String {
         var id = Constants.alertTitleBlock + String(county.countyID)
-        if county.hasExtras() {
-            let extras = county.countyExtras
-            if let nb = extras?.newBorns {
-                id.append("\n" + Constants.alertNewborns + String(nb))
-            }
-            if let pop = extras?.population {
-                id.append("\n" + Constants.alertPopulation + String(pop))
-            }
+        let extras = county.countyExtras
```

Comment:
This is ok, but the answer was: `if let extras = county.countyExtras { .... }` since you were checking `!= nil` and then using `extras?.XXX` to get the value, which didn't make sense. Using `if let` does the unwrapping for you and the check that there are extras at the same time.

Reply:
Makes way more sense 🤦‍♂️. Change made, thanks